core/sr-primitives/src/generic/digest.rs

```diff
@@ -71,14 +71,19 @@ pub enum DigestItem<Hash, AuthorityId, SealSignature> {
 	/// block. It is created for every block iff runtime supports changes
 	/// trie creation.
 	ChangesTrieRoot(Hash),
```

Comment:
We should be able to remove the `#[allow(deprecated)]` attribute on this enum now.

Reply:
Done.